src/Query/User/Model/ScanResult/ResultContent.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Query\User\Model\ScanResult;
+
+use Buddy\Repman\Query\User\Model\ScanResult\ResultContent\Advisory;
+use Buddy\Repman\Query\User\Model\ScanResult\ResultContent\Dependency;
+use Buddy\Repman\Query\User\Model\ScanResult\ResultContent\LockFileResult;
+
+final class ResultContent
+{
+    private string $json;
+
+    public function __construct(string $json)
+    {
+        $this->json = $json;
+    }
+
+    public function html(): string
+    {
+        return $this->formatContent();
+    }
+
+    public function htmlSimple(): string
```

Comment:
I would like to rename this method and maybe we can remove html from readl model?
I know it simplifies, but it's not their responsibility to format html.